src/ui/components/delegation/TrackSelect.tsx

```diff
@@ -7,11 +7,21 @@ interface ICheckBoxProps {
   title?: string;
   checked?: boolean;
   onChange?: React.ChangeEventHandler<HTMLInputElement>;
+  background?: boolean;
```

Comment:
I'll remove this and add a className to the props and pass the custom background style as a className extension. There are examples in the lib (e.x. Buttons)

Reply:
i'd say trackSelect will always be on background, and probably others that evolve from this too - 
<img width="1092" alt="image" src="https://user-images.githubusercontent.com/60112552/223070549-d7414912-4f37-483a-b772-e7f8c44675f4.png">
but totally your call!

Reply:
Both works. No string opinion.